src_nbgl/ui_sign_712.c

```diff
@@ -48,18 +48,10 @@ static bool display_review_page(uint8_t page, nbgl_pageContent_t *content) {
             break;
 
         case 1:
-            switch (ui_712_next_field()) {
-                case EIP712_NO_MORE_FIELD:
-                    ui_712_switch_to_sign();
-                    ret = true;
```

Comment:
This `ret == true` was never used right?
Your code is equivalent but I think there is a deeper problem here..

Reply:
cc @apaillier-ledger

Reply:
The previous code was setting `ret = true`, but just behind, we had `__attribute__((fallthrough))`, which set `ret = false`.
That means the `ret = true` was useless, and I have simplified the code.

Reply:
Ok but this is a bug (present previously but still a bug), the function should return `true` if it succeeds

Reply:
Not necessarily a bug since everything is already behaving (functionally speaking) well.
This return value controls if NBGL should refresh the screen if I recall correctly. :thinking: